abb_experimental.rosinstall

```diff
@@ -0,0 +1,5 @@
+#Temporary until PR accepted by https://github.com/ros-industrial/
+- git: {local-name: abb, uri: 'https://github.com/AustinDeric/abb.git', version: kinetic-devel}
+
+- git: {local-name: industrial_core, uri: 'https://github.com/ros-industrial/industrial_core.git', version: kinetic-devel}
```

Comment:
I can update the readme for the rosdep method if you want.  We will still have to use the rosinstall method to get the abb package.  So in effect we will have to do rosinstall + rosdep rather than just rosinstall.